src/lib/PreviewUI.js

```diff
@@ -48,6 +48,9 @@ class PreviewUI {
     /** @property {HTMLElement} - Preview container element which houses the sidebar and content */
     previewContainer;
 
+    /** @property {HTMLElement} - Preview container element which houses the custom-logo */
+    customLogoEl;
```

Comment:
It doesn't seem like `customLogoEl` is referenced anywhere else, could the `alt` attribute be set as part of the `setupHeader` logic without introducing this class level property?

Reply:
the problem are these lines:

```
if (options.header !== 'none') {
     this.setupHeader(options.header, options.logoUrl);
```

if there's no `options.header` then `setupHeader` is not call,  so `customLogoEl` exists but it is hidden because the class `CLASS_HIDDEN`

however even if its hidden is an error for a11y, so that's why you need to be sure to always set the `alt` attribute

Reply:
@tille, can you please clarify how this affects the end user? The `CLASS_HIDDEN` class, `bp-is-hidden`, sets `display: none` on the element to which it's applied, which essentially removes it from the DOM as it is not accessible to the user.